gemfiles/Gemfile-rails-5-2

```diff
@@ -1,4 +1,6 @@
 source 'https://rubygems.org'
 gemspec path: '..'
 
-gem 'actionpack', '~> 5.2.0'
\ No newline at end of file
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}.git" }
+
+gem 'actionpack', '~> 5.2.0', github: 'rails/rails', tag: '5-2-stable'
```

Comment:
Shouldn't this be `branch:` instead of `tag:`?

Reply:
Ah, you're absolutely right!

(They are, technically, aliases, but here, we absolutely mean _branch_.)

Reply:
Oh, I didn't know they were aliases. But yeah, this is clearer.

Reply:
`ref` is the most technical alias.